.packit.yaml

```diff
@@ -0,0 +1,24 @@
+# See the documentation for more information:
+# https://packit.dev/docs/configuration/
+
+specfile_path: criu.spec
+
+# add or remove files that should be synced
+synced_files:
+    - criu.spec
+    - .packit.yaml
+    - criu-tmpfiles.conf
+
+# name in upstream package repository/registry (e.g. in PyPI)
+upstream_package_name: criu
+# downstream (Fedora) RPM package name
+downstream_package_name: criu
+actions:
+    post-upstream-clone: "wget https://src.fedoraproject.org/rpms/criu/raw/rawhide/f/criu.spec -O criu.spec; https://src.fedoraproject.org/rpms/criu/raw/rawhide/f/criu-tmpfiles.conf -O criu-tmpfiles.conf"
```

Comment:
these are not invoked in shell, that's why the build failed, you can change it to:
```
actions:
  post-upstream-clone:
  - "wget https://src.fedoraproject.org/rpms/criu/raw/rawhide/f/criu.spec -O criu.spec"
  - "wget https://src.fedoraproject.org/rpms/criu/raw/rawhide/f/criu-tmpfiles.conf -O criu-tmpfiles.conf"
```

Running commands in shell requires wrapping them in `bash -c "..."` which is not needed here.